include/zephyr/bluetooth/ead.h

```diff
@@ -0,0 +1,147 @@
+/* Copyright (c) 2023 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include <zephyr/kernel.h>
+
+#include <zephyr/bluetooth/bluetooth.h>
+
+/** Additional Authenticated Data size in bytes */
+#define BT_EAD_AAD_SIZE	       1
+/** Randomizer size in bytes */
+#define BT_EAD_RANDOMIZER_SIZE 5
+/** nonce size in bytes */
+#define BT_EAD_NONCE_SIZE      13
+/** Key size in bytes */
+#define BT_EAD_KEY_SIZE	       16
+/** Initialisation Vector size in bytes */
+#define BT_EAD_IV_SIZE	       8
+/** MIC size in bytes */
+#define BT_EAD_MIC_SIZE	       4
+
+/**
+ * @brief Encrypt and authenticate the given advertising data. The resulting
+ * data in @p encrypted_payload will look like that:
+ * - Randomizer is added in the 5 first bytes;
```

Comment:
Should we do something like 
```suggestion
 * - Randomizer is added in the @ref BT_EAD_RANDOMIZER_SIZE first bytes;
```
for these?